jenkins/nightly_tar.sh

```diff
@@ -57,7 +61,20 @@ docker \
       --selenium-driver-args headless \
       --selenium-driver-args no-sandbox \
       --remote-host $(host nas02.arangodb.biz |sed "s;.* ;;") \
-      $force_arg --git-version $GIT_VERSION $@
+      $force_arg --git-version $GIT_VERSION $@ &
+docker_pid=$!
+
+sleep 2
+
+PID=$(ps -eaf | \
+          grep -v docker |\
+          grep -v grep |\
+          grep full_download_upgrade_test.py |\
+          sed "s;root *\([0-9]*\) .*;\1;")
+
+sudo /usr/sbin/killsnoop.bt -p $PID >  $(pwd)/test_dir/kill_snoop.txt
```

Comment:
Will it catch `kill()` inside docker container being launched outside?

Reply:
yes. `--pid=host` enables us to have meaningfull PIDs inside and outside the docker container.

Reply:
https://github.com/brendangregg/perf-tools/issues/106 maybe somebody knows better than me.